docs/RELEASE_ISSUE_TEMPLATE.md

```diff
@@ -10,154 +12,217 @@
+    - [ ] Pin the topic globally so that it stays at the top of the category.
+    - [ ] If there is no more important banner currently set on Discourse (e.g. IPFS Camp announcement), make the topic into a banner.
+    - [ ] Check if Discourse post was automatically copied to:
+      - [ ] IPFS Discord #ipfs-chatter
+      - [ ] FIL Slack #ipfs-chatter
+      - [ ] Matrix
+  - [ ] Add a link from release notes to Discuss post (like we did here: https://github.com/ipfs/kubo/releases/tag/v0.15.0)
+  - [ ] Update the draft PR created for [interop](https://github.com/ipfs/interop) to use the new release and mark it as ready for review.
+  - [ ] Update the draft PR created for [IPFS Desktop](https://github.com/ipfs-shipyard/ipfs-desktop) to use the new release and mark it as ready for review.
+  - [ ] Update docs
+    - [ ] Run https://github.com/ipfs/ipfs-docs/actions/workflows/update-on-new-ipfs-tag.yml to generate a PR to the docs repo
+    - [ ] Merge the auto-created PR in https://github.com/ipfs/ipfs-docs/pulls ([example](https://github.com/ipfs/ipfs-docs/pull/1263))
+  - [ ] Get the blog post created and shared
+    - [ ] Submit a request for blog post creation using [the form](https://airtable.com/shrNH8YWole1xc70I).
+      - Notify marketing in #shared-pl-marketing-requests about the blog entry request (since the form tends to go to spam).
+      - Don't mark this as done until the blog entry is live.
+    - [ ] Share the blog post
+      - [ ] Twitter (request in Filecoin Slack channel #shared-pl-marketing-requests)
+      - [ ] [Reddit](https://reddit.com/r/ipfs)
+- [ ] **Stage 6 - Post-Release**
   - [ ] Merge the `release` branch back into `master`, ignoring the changes to `version.go` (keep the `-dev` version from master).
   - [ ] Create an issue using this release issue template for the _next_ release.
-  - [ ] Make sure any last-minute changelog updates from the blog post make it back into the CHANGELOG.
-  - [ ] Mark PR draft created for IPFS Desktop as ready for review.
-  
+  - [ ] Close this release issue.
+
 ## ⁉️ Do you have questions?
 
 The best place to ask your questions about IPFS, how it works and what you can do with it is at [discuss.ipfs.io](http://discuss.ipfs.io). We are also available at the `#ipfs` channel on Freenode, which is also [accessible through our Matrix bridge](https://riot.im/app/#/room/#freenode_#ipfs:matrix.org).
 
-## Release improvements for next time
-
-< Add any release improvements that were observed this cycle here so they can get incorporated into future releases. >
-
-## Items for a separate comment
-
-< Do these as a separate comment to avoid the main issue from getting too large and checkbox updates taking too long. >
-
-### Changelog
-
-< changelog generated by bin/mkreleaselog > (add it to a separated comment if it is too big)
-
-### ❤️ Contributors
-
-< list generated by bin/mkreleaselog >
```

Comment:
I propose to comment with a link to the final release on the release issue instead. The final release will already have this information.